plotjuggler_plugins/DataLoadRlog/rlog_parser.cpp

```diff
@@ -60,16 +60,10 @@ bool RlogMessageParser::parseMessageCereal(capnp::DynamicStruct::Reader event)
 
   uint64_t last_sec = event.get("logMonoTime").as<uint64_t>();
   double time_stamp = (double)last_sec / 1e9;
-  if (event.has("can")) {
-    return parseCanMessage("/can", event.get("can").as<capnp::DynamicList>(), time_stamp, last_sec);
-  } else if (event.has("sendcan")) {
-    return parseCanMessage("/sendcan", event.get("sendcan").as<capnp::DynamicList>(), time_stamp, last_sec);
-  } else {
-    return parseMessageImpl("", event, time_stamp, true);
-  }
+  return parseMessageImpl("", event, time_stamp, last_sec, true);
 }
 
-bool RlogMessageParser::parseMessageImpl(const std::string& topic_name, capnp::DynamicValue::Reader value, double time_stamp, bool is_root)
+bool RlogMessageParser::parseMessageImpl(const std::string& topic_name, capnp::DynamicValue::Reader value, double time_stamp, uint64_t last_sec, bool is_root)
```

Comment:
Decided to pass last_sec into the function, as dividing the uint64_t by 1e9, then multiplying by 1e9 can lead to errors of 1 ns. Not sure if it matters too much but just didn't want to worry about verifying the ordering of all CAN messages don't change